modules/URL/ConferenceUrl.js

```diff
@@ -67,7 +67,14 @@ export default class ConferenceUrl {
      * Reloads the conference using original URL with all of the parameters.
      */
     reload() {
-        logger.info("Reloading the conference using URL: " + this.originalURL);
-        replace(this.originalURL);
+        logger.info(`Reloading the conference using URL: ${this.originalURL}`);
+
+        // Check if we are in an iframe and reload with the reload() utility
+        // because replace() is not working on an iframe.
+        if(window.self !== window.top) {
+            reload();
```

Comment:
I know we have agreed that importing functions directly is cleaner, but in this case the code looks misleading and it's not clear enough that this reload method comes from util/helpers. Maybe I'm just not used to it, but I'm sharing an observation.